fe/fe-core/src/main/java/com/starrocks/catalog/OlapTable.java

```diff
@@ -196,7 +196,8 @@ public enum OlapTableState {
          * The query plan which is generate during this state is invalid because the meta
          * during the creation of the logical plan and the physical plan might be inconsistent.
         */
-        UPDATING_META
+        UPDATING_META,
+        OPTIMIZE
```

Comment:
are there any compatibility issues with upgrade or rollback when optimize job is running?

Reply:
it should be compatibility with upgrade or downgrade since replay will update table state.